docs/general/guides/communications/communications.md

```diff
@@ -0,0 +1,114 @@
+# Communications
+
+You can send custom messages to voters at any time during the electoral process, from
+the election page in the admin console. It can be used to send authentication codes for
+elections that require it, or to send communications announcing the different stages
+of the election.
+
+Those messages will be sent through sms or email depending on the authentication 
+method. Each voter will receive a tailored message with the template variables replaced
+with their values. 
+
+You can send messages either send bulk messages to the whole census or send messages
+to specific lists of users. This is further explained below.
+
+## Template variables
+
+Templates are text messages that include template variables. Each voter will receive a
+tailored message with the template variables replaced with their values. Depending on
+the authentication method, the message will be delivered as an email or as an SMS.
+
+For authentication methods such as `email`, `email-otp`, or `email-and-password` the
+message will be an email. In those cases, the email title and email text body are
+templates. If [html messages](../../reference/election-creation-json#census-config-html_message)
+are enabled in the deployment, the HTML message is also a template. For authentication
+methods such as `sms` or `sms-otp`, the SMS text body is a template. Furthermore, you can also use an
+[extra field](../../reference/election-creation-json#census-extra_fields) of type `otp-code`
+to send authentication messages for any authentication method.
+
+In templates, variables are identified surrounded by two `_` characters and always
+in upper case. For example the variable `url` would appear as `__URL__`.
+
+The allowed template variables are:
+- `__URL__`: This is the voter authentication URL specific for the voter but
+not containing the voter authentication code, which the voter will have to fill
+out manually.
+- `__URL2__`: This is the voter authentication URL containing the 
+both the email/sms of the voter and the voter authentication code. If no other
+[extra_field](../../reference/election-creation-json#census-extra_fields) is required during authentication, entering  
+in the `__URL2__` URLs allows voters to authenticate without having to fill out 
+any web form. It's easier, but also more risky because anyone with this link 
+could use it to authenticate.
+- `__CODE__`: This is the authentication code. Each time the authentication 
+codes are sent to a voter, a new code is generated and any old codes are 
+disabled.
```

Comment:
please below this list add a `::::note` (see https://docusaurus.io/docs/next/markdown-features/admonitions ) mentioning that.. even if the message to voter(s) does not contain `__URL2__` nor `__CODE__`, a new code will be generated nevertheless. We will be changing this in the future soon, but anyway right now it is this way.